components/gfx/platform/macos/font_context.rs

```diff
@@ -2,7 +2,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#[derive(Clone, Debug, MallocSizeOf)]
+use malloc_size_of::{MallocSizeOf, MallocSizeOfOps};
+
+#[derive(Clone, Debug)]
```

Comment:
I don't understand this, why is `#[derive()]` not usable anymore?

Reply:
`#[derive()]` is still usable but it is only once used in the whole crate and only on MacOS. (I didn't first notice because on Linux it built fine.) Should I add the dependency back and use `#[derive()]` again? Btw are there platform specific dependencies?